### PR TITLE
Recsys fix

### PR DESCRIPTION
**Pull request by ceteri**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,8 @@
 gym >= 0.17.2
-paretoset >= 1.1.2
 numpy >= 1.18.5
 pandas
 requests
-pytorch
```
Comment (deanwampler-domino) [inline requirements.txt:6]: I noticed this distinction before. It's actually pytorch if you do a conda install. I don't think I ever fixed this or tested in in the pip install with this requirements.txt file. Thanks for the catch.

File: requirements.txt [hunk 2]
```diff
@@ -18,7 +17,6 @@ jupyterlab
 jupyter-server-proxy
 beautifulsoup4
 lxml
-setproctitle
```
Comment (deanwampler-domino) [inline requirements.txt:21]: I don't know why I added this, but if it appears to work without it, great.

**Review by deanwampler-domino — APPROVED** [review]
LGTM.

File: requirements.txt (hunk 2 above)
Comment (ceteri) [reply requirements.txt:21]: Thank you!

**Pull request merged**